.gitignore

```diff
@@ -15,3 +15,8 @@ Gemfile-*.lock
 .ruby-version
 .DS_Store
 /test.rb
+
+# Native extension binaries
+lib/**/*.bundle
+lib/**/*.so
+lib/**/*.o
```

Comment:
```
lib/**/*.dll # windows
lib/**/*.dylib # osx
```

Reply:
hm, on osx I have bundle and we don't support Windows but I will add just in case